src/components/Cards/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { useFetch } from "../../utils/hooks/useFetch";
+import { formatDate, generateTimeText } from "../../utils/hooks/date";
+import thumbnail from "../../assets/thumbnail.svg";
+import styles from "./index.module.css";
+
+function Cards(props) {
+  const CardData = useFetch(props.url);
+
+  return (
+    <div className={styles.card_grid_container}>
+      {CardData ? (
+        CardData.folder.links.map((link) => (
+          <a href={link.url}>
```

Comment:
map 하시면 key 를 꼭 넣어주셔야합니다!
```suggestion
          <a href={link.url} key={link.url}>
```